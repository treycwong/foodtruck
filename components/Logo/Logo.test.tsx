import { render } from "@/test-utils";

import { Logo } from "./Logo";

describe("Logo test cases", () => {
  it("Render check", () => {
    const { asFragment } = render(<Logo>FoodTruck</Logo>);
    expect(asFragment()).toMatchSnapshot();
  });
  it("Render with custom size", () => {
    const { asFragment } = render(<Logo size={10}>FoodTruck</Logo>);
    expect(asFragment()).toMatchSnapshot();
  });
});
