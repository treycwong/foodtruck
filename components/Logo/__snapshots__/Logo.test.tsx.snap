// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo test cases Render check 1`] = `
<DocumentFragment>
  <header
    class="css-1o9x2ib"
  >
    FoodTruck
  </header>
</DocumentFragment>
`;

exports[`Logo test cases Render with custom size 1`] = `
<DocumentFragment>
  <header
    class="css-1lpr8zk"
    size="10"
  >
    FoodTruck
  </header>
</DocumentFragment>
`;
